rails/README.md

```diff
@@ -15,6 +15,9 @@
 - Order i18n translations alphabetically by key name.
 - Order model contents: constants, macros, public methods, private methods.
 - Put application-wide partials in the [`app/views/application`] directory.
+- Put classes (Service Objects/Interactors/et al) that may later be extracted
```

Comment:
Thanks for distilling this! I have a few thoughts:

- Should this be two bullets? (One about `lib/` and one about `app/jobs`)
-  Maybe something like "Use `lb/` for code that will later be extracted to a gem" to describe `lib`? I think it could be anything generic, it's hard to make a finite list. My opinion is that `lib/` shouldn't have app-specific code.